spec/integration/inventory2_spec.rb

```diff
@@ -496,4 +496,50 @@ def fact_plan(name = 'facts_test')
       expect(result[0]).to include('status' => 'success', 'target' => conn[:host])
     end
   end
+
+  context 'with prompt inventory_config_lookups', ssh: true do
+    let(:inventory) do
+      {
+        version: 2,
+        targets: [{
+          name: 'target-1',
+          config: {
+            transport: 'ssh',
+            ssh: {
+              password: { _plugin: 'prompt', message: 'password please' },
+              user: conn[:user],
+              host: conn[:host],
+              port: conn[:port],
+              'host-key-check': false
+            }
+          }
+        },
+                  {
+                    name: 'target-2',
```

Comment:
Tired of fighting rubocop on this. For some reason it insists this is the correct indentation.

Reply:
finally won the battle and got reasonable indentation.